src/factories/DisplayListOverrides.h

```diff
@@ -3,6 +3,36 @@
 #include <cstdint>
 #include "DisplayListFactory.h"
 
+typedef struct {
+    uint32_t w0;
+    uint32_t w1;
+} Words;
+
+typedef union {
+    Words words;
+    long long int	force_structure_alignment;
+} Gfx;
+
+typedef struct {
+    short		    ob[3];
```

Comment:
shouldn't these be int16_t ? or whatever?

Reply:
Yeah but both works, in the end they are the same